apps/zotonic_core/include/zotonic_deprecated.hrl

```diff
@@ -1,31 +1,25 @@
 %% @doc Deprecated defintions.
 %%      These will be removed in a future update
 
-
 %% Record used for transporting data between the user-agent and the server.
 %% This was part of the transport system, before the Cotonic/MQTT integration.
--record(z_msg_v1, {
-    qos = 0 :: 0 | 1 | 2,
-    dup = false :: boolean(),
-    msg_id :: undefined | binary(),
-    timestamp :: undefined | pos_integer(),
-    content_type = ubf :: text | javascript | json | form | ubf | atom() | binary(),
-    delegate = postback :: postback | mqtt | atom() | binary(),
-    push_queue = page :: page | session | user,
-
-    % Set by transports from user-agent to server
-    ua_class = undefined :: ua_classifier:device_type() | undefined,
-    session_id :: undefined | binary(),
-    page_id :: undefined | binary(),
-
-    % Payload data
-    data :: any()
-}).
-
+-record(z_msg_v1,
+        {qos = 0 :: 0 | 1 | 2,
```

Comment:
I don't like this `{` on this position, as either it is not a  tab stop, or the list below is not a tab stop.

That is why I normally add the `{` at the end of the previous line.